### PR TITLE
Adds support for showing the metadata and trending Artists to a Gene VC

### PR DESCRIPTION
**Pull request by orta**
Gap at the top is for follow button 

screen shot 2016-09-30 at 15 08 40

screen shot 2016-09-30 at 15 08 43

### CODE REVIEW

File: data/schema.graphql [hunk 1]
```diff
@@ -315,6 +315,8 @@ type ArtworkContextAuction {
   start_at(format: String, timezone: String, convert_to_utc: Boolean): String
```
Comment (orta) [inline data/schema.graphql]: needed to update the schema for description

File: externals/metaphysics [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 1541a0d42480bd95aee87898200a8ad88e086a45
+Subproject commit 6ecc7b5a8bc124dd807364d102b24e52a51e5ccc
```
Comment (orta) [inline externals/metaphysics]: ^ just a notice

File: lib/containers/gene.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ class Gene extends React.Component {
   };
 
   componentWillMount() {
-    this.state = { selectedTabIndex: 0 }
+    this.state = { selectedTabIndex: 1 }
```
Comment (orta) [inline lib/containers/gene.js]: defaults to the about info ATM, as there's no artworks being downloaded

**Comment by alloy** [comment]
Looking good!

There’s a bunch of overlap with components on the artist view, will those be sharable in the future?

File: lib/components/artist/about.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ import { View, StyleSheet } from 'react-native'
 
 import Biography from './biography'
 import Articles from './articles'
-import RelatedArtists from './related_artists'
+import RelatedArtists from '../related_artists'
```
Comment (orta) [inline lib/components/artist/about.js]: now shared between components 
screen shot 2016-09-30 at 16 07 54

File: lib/containers/__tests__/gene-tests.js [hunk 5]
```diff
@@ -17,7 +17,21 @@ it('looks like expected', () => {
...
+            name: 'Artist name',
+            counts: {
+              for_sale_artworks: 1,
+              artworks: 2
+            },
+            image: {
+              large_version: ''
+            }
+          }
+        ]
```
Comment (orta) [inline lib/containers/__tests__/gene-tests.js]: need to fill up the data we would get from metaphysics

File: tsconfig.json [hunk 6]
```diff
@@ -5,6 +5,7 @@
     "exclude": [
         "node_modules",
         "Pod/Assets",
-        "Example/Build"
+        "Example/Build",
+        "externals/"
```
Comment (orta) [inline tsconfig.json]: no getting type data from metaphysics

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">No CHANGELOG added.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">The following files do not have tests: <a href="https://github.com/artsy/emission/blob/466bdef69fdbcdceb3b35242370edc328605ae1e/lib/__mocks__/react-relay.js">lib/<strong>mocks</strong>/react-relay.js</a>, <a href="https://github.com/artsy/emission/blob/466bdef69fdbcdceb3b35242370edc328605ae1e/lib/components/artist/about.js">lib/components/artist/about.js</a>, <a href="https://github.com/artsy/emission/blob/466bdef69fdbcdceb3b35242370edc328605ae1e/lib/components/artist/related_artists/index.js">lib/components/artist/related_artists/index.js</a>, <a href="https://github.com/artsy/emission/blob/466bdef69fdbcdceb3b35242370edc328605ae1e/lib/components/artist/related_artists/related_artist.js">lib/components/artist/related_artists/related_artist.js</a>, <a href="https://github.com/artsy/emission/blob/466bdef69fdbcdceb3b35242370edc328605ae1e/lib/components/gene/about.js">lib/components/gene/about.js</a>, <a href="https://github.com/artsy/emission/blob/466bdef69fdbcdceb3b35242370edc328605ae1e/lib/components/gene/about_gene.js">lib/components/gene/about_gene.js</a>, <a href="https://github.com/artsy/emission/blob/466bdef69fdbcdceb3b35242370edc328605ae1e/lib/components/gene/biography.js">lib/components/gene/biography.js</a>, <a href="https://github.com/artsy/emission/blob/466bdef69fdbcdceb3b35242370edc328605ae1e/lib/components/gene/header.js">lib/components/gene/header.js</a>, <a href="https://github.com/artsy/emission/blob/466bdef69fdbcdceb3b35242370edc328605ae1e/lib/components/related_artists/index.js">lib/components/related_artists/index.js</a> &amp; <a href="https://github.com/artsy/emission/blob/466bdef69fdbcdceb3b35242370edc328605ae1e/lib/components/related_artists/related_artist.js">lib/components/related_artists/related_artist.js</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by orta** [comment]
Yep, we're green now

**Comment by orta** [comment]
Alright @mennenia you're up for review 👯

**Comment by mennenia** [comment]
Looks great!

**Comment by rogeliog** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>No Changelog changes!</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by orta** [comment]
Hah oops, looks like it's Danger JS is still falling back to my FakeCI implementation

**Comment by benkraus** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Neither the title nor body of the pull request reference a JIRA ticket.</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by smromain** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Running your Dangerfile has Failed
  </td>
    </tr>
  </tbody>
</table>




## Danger has errored

Error: Error

```sh
Error: ENOENT: no such file or directory, open 'lib/components/gene/about.js'
    at Object.fs.openSync (fs.js:557:18)
    at Object.fs.readFileSync (fs.js:467:33)
    at dangerfile.js:86:20
    at Array.filter (native)
    at checkForAmigo (dangerfile.js:85:32)
    at Object.<anonymous> (dangerfile.js:119:1)
    at Runtime._execModule (/Users/sromain/dev/agent-inbox/node_modules/jest-runtime/build/index.js:513:13)
    at Runtime.requireModule (/Users/sromain/dev/agent-inbox/node_modules/jest-runtime/build/index.js:329:14)
    at /Users/sromain/dev/agent-inbox/node_modules/danger/distribution/runner/DangerfileRunner.js:133:33
    at ensureCleanDangerfile (/Users/sromain/dev/agent-inbox/node_modules/danger/distribution/runner/DangerfileRunner.js:173:5)
```


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by rogeliog** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :tada:, this worked @orta
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by genies** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  /mnt/c/Development/Projects/GenieBelt/genieplanner-html/node_modules/danger/distribution/commands/danger-runner.js,https://github.com/GenieBelt/genieplanner-html/pull/247` failed.
  </td>
    </tr>
  </tbody>
</table>



### Log


```sh

```

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by sweatcoin** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Danger failed to run `dangerfile.js`.
  </td>
    </tr>
  </tbody>
</table>



## Error ReferenceError
```
errorCount is not defined
ReferenceError: errorCount is not defined
    at Object.<anonymous> (dangerfile.js:1:793)
    at Module._compile (module.js:570:32)
    at requireFromString (/Users/nvzn/Developer/Sweatcoin/sweatcoin-ios/sweatcoin-react-native/node_modules/require-from-string/index.js:28:4)
    at Object.<anonymous> (/Users/nvzn/Developer/Sweatcoin/sweatcoin-ios/sweatcoin-react-native/node_modules/danger/distribution/runner/runners/inline.js:102:21)
    at step (/Users/nvzn/Developer/Sweatcoin/sweatcoin-ios/sweatcoin-react-native/node_modules/danger/distribution/runner/runners/inline.js:32:23)
    at Object.next (/Users/nvzn/Developer/Sweatcoin/sweatcoin-ios/sweatcoin-react-native/node_modules/danger/distribution/runner/runners/inline.js:13:53)
    at /Users/nvzn/Developer/Sweatcoin/sweatcoin-ios/sweatcoin-react-native/node_modules/danger/distribution/runner/runners/inline.js:7:71
    at __awaiter (/Users/nvzn/Developer/Sweatcoin/sweatcoin-ios/sweatcoin-react-native/node_modules/danger/distribution/runner/runners/inline.js:3:12)
    at Object.runDangerfileEnvironment (/Users/nvzn/Developer/Sweatcoin/sweatcoin-ios/sweatcoin-react-native/node_modules/danger/distribution/runner/runners/inline.js:69:12)
    at Object.<anonymous> (/Users/nvzn/Developer/Sweatcoin/sweatcoin-ios/sweatcoin-react-native/node_modules/danger/distribution/commands/danger-runner.js:81:55)
```
### Dangerfile
```
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------^
```
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by robothanx** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Danger failed to run `dangerfile.js`.
  </td>
    </tr>
  </tbody>
</table>



## Error SyntaxError
```
Unexpected token import
dangerfile.js:2
import { istanbulCoverage } from "danger-plugin-istanbul-coverage";
^^^^^^

SyntaxError: Unexpected token import
    at new Script (vm.js:51:7)
    at createScript (vm.js:136:10)
    at Object.runInThisContext (vm.js:197:10)
    at Module._compile (module.js:613:28)
    at requireFromString (/Users/marc/Development/Thanx/merchant-ui/node_modules/require-from-string/index.js:28:4)
    at Object.<anonymous> (/Users/marc/Development/Thanx/merchant-ui/node_modules/danger/distribution/runner/runners/inline.js:132:34)
    at step (/Users/marc/Development/Thanx/merchant-ui/node_modules/danger/distribution/runner/runners/inline.js:32:23)
    at Object.next (/Users/marc/Development/Thanx/merchant-ui/node_modules/danger/distribution/runner/runners/inline.js:13:53)
    at /Users/marc/Development/Thanx/merchant-ui/node_modules/danger/distribution/runner/runners/inline.js:7:71
    at new Promise (<anonymous>)
```
### Dangerfile
```
---------^
```
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by robothanx** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Danger failed to run `dangerfile.js`.
  </td>
    </tr>
  </tbody>
</table>



## Error SyntaxError
```
Unexpected token import
dangerfile.js:1
(function (exports, require, module, __filename, __dirname) { import { istanbulCoverage } from "danger-plugin-istanbul-coverage";
                                                              ^^^^^^

SyntaxError: Unexpected token import
    at new Script (vm.js:51:7)
    at createScript (vm.js:136:10)
    at Object.runInThisContext (vm.js:197:10)
    at Module._compile (module.js:613:28)
    at requireFromString (/Users/marc/Development/Thanx/merchant-ui/node_modules/require-from-string/index.js:28:4)
    at Object.<anonymous> (/Users/marc/Development/Thanx/merchant-ui/node_modules/danger/distribution/runner/runners/inline.js:132:34)
    at step (/Users/marc/Development/Thanx/merchant-ui/node_modules/danger/distribution/runner/runners/inline.js:32:23)
    at Object.next (/Users/marc/Development/Thanx/merchant-ui/node_modules/danger/distribution/runner/runners/inline.js:13:53)
    at /Users/marc/Development/Thanx/merchant-ui/node_modules/danger/distribution/runner/runners/inline.js:7:71
    at new Promise (<anonymous>)
```
### Dangerfile
```
---------^
```
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by bynder-gocd** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please add the JIRA issue key to the PR title (e.g. SETTINGS,DEIMOS,PHOBOS,COCO,PROC,MARS,DATA,BA,INT,THOR-123)
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  :exclamation: Big PR (1)
  </td>
    </tr>
  </tbody>
</table>



> (1): Pull Request size seems relatively large.
      If Pull Request contains multiple changes, split each into separate PR will helps faster, easier review.

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by alloy** [comment]
Can’t restore @mennenia as the assigned person because she’s no longer in the org, so leaving this message for posterity.

**Comment by AntoineDoubovetzky** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Danger failed to run `dangerfile.js`.
  </td>
    </tr>
  </tbody>
</table>



# Coverage

| File                                           | Branches                                              | Statements                                            |
| ---------------------------------------------- | ----------------------------------------------------- | ----------------------------------------------------- |
|                                                |                                                       |                                                       |
| :heavy_plus_sign: **NEW FILES**                |                                                       |                                                       |
|                                                |                                                       |                                                       |
| :file_folder: **lib**                          |                                                       |                                                       |
| ↦ components/gene/about.js                     |  0% |  0% |
| ↦ components/gene/biography.js                 |  0% |  0% |
|                                                |                                                       |                                                       |
| :pencil2: **MODIFIED FILES**                   |                                                       |                                                       |
|                                                |                                                       |                                                       |
| :file_folder: **lib**                          |                                                       |                                                       |
| ↦ components/artist/about.js                   |  0% |  0% |
| ↦ components/gene/header.js                    |  0% |  0% |
| ↦ components/related_artists/index.js          |  0% |  0% |
| ↦ components/related_artists/related_artist.js |  0% |  0% |
| ↦ containers/gene.js                           |  0% |  0% |

## Error Error
```
ENOENT: no such file or directory, open 'lib/components/gene/about.js'
Error: ENOENT: no such file or directory, open 'lib/components/gene/about.js'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.readFileSync (fs.js:551:33)
    at checkForFlow (/Users/antoinedoubovetzky/Projects/dygest/dygest-app/node_modules/@bam.tech/danger-plugin-flow/dist/index.js:53:24)
    at Array.filter (<anonymous>)
    at /Users/antoinedoubovetzky/Projects/dygest/dygest-app/node_modules/@bam.tech/danger-plugin-flow/dist/index.js:66:45
    at Generator.next (<anonymous>)
    at /Users/antoinedoubovetzky/Projects/dygest/dygest-app/node_modules/@bam.tech/danger-plugin-flow/dist/index.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/antoinedoubovetzky/Projects/dygest/dygest-app/node_modules/@bam.tech/danger-plugin-flow/dist/index.js:3:12)
    at flow (/Users/antoinedoubovetzky/Projects/dygest/dygest-app/node_modules/@bam.tech/danger-plugin-flow/dist/index.js:61:12)
```
### Dangerfile
```
---------------------^
```
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by eltonlauhk01** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please assign someone to merge this PR, and optionally include people who should review.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by sweatcoin** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  There are 1 lint violations:

- ERROR: /Users/nvzn/Developer/sweatcoin-ios/sweatcoin-react-native/js/__tests__/OnboardingRenderer.test.tsx[34, 1]: A maximum of 3 classes per file are allowed.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  There is no analytics for this feature. Are you sure you have everything in place?
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by mohamadarabi** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  node` failed.
  </td>
    </tr>
  </tbody>
</table>



### Log


```sh
Error:  { SyntaxError: /Users/mohamadarabi/c3/c3base/dangerfile.js: Unexpected token, expected "," (94:4)

  92 |       plato.inspect(files, outputDir, options, callback);
  93 |     }
> 94 |     async eslint (danger, comment) {
     |     ^
  95 |         const filesToLint = danger.git.created_files.concat(danger.git.modified_files).filter(function (file) {
  96 |             return file.endsWith('.js') || file.endsWith('.jsx');
  97 |         });
    at Parser.raise (/Users/mohamadarabi/c3/c3base/node_modules/@babel/parser/lib/index.js:4028:15)
    at Parser.unexpected (/Users/mohamadarabi/c3/c3base/node_modules/@babel/parser/lib/index.js:5359:16)
    at Parser.expect (/Users/mohamadarabi/c3/c3base/node_modules/@babel/parser/lib/index.js:5347:28)
    at Parser.parseObj (/Users/mohamadarabi/c3/c3base/node_modules/@babel/parser/lib/index.js:6838:14)
    at Parser.parseExprAtom (/Users/mohamadarabi/c3/c3base/node_modules/@babel/parser/lib/index.js:6464:21)
    at Parser.parseExprSubscripts (/Users/mohamadarabi/c3/c3base/node_modules/@babel/parser/lib/index.js:6081:21)
    at Parser.parseMaybeUnary (/Users/mohamadarabi/c3/c3base/node_modules/@babel/parser/lib/index.js:6060:21)
    at Parser.parseExprOps (/Users/mohamadarabi/c3/c3base/node_modules/@babel/parser/lib/index.js:5945:21)
    at Parser.parseMaybeConditional (/Users/mohamadarabi/c3/c3base/node_modules/@babel/parser/lib/index.js:5917:21)
    at Parser.parseMaybeAssign (/Users/mohamadarabi/c3/c3base/node_modules/@babel/parser/lib/index.js:5864:21)
  pos: 3630,
  loc: Position { line: 94, column: 4 },
  code: 'BABEL_PARSE_ERROR' }
danger-results://var/folders/qq/bf0tqkhn5pl1gsm0bzr16m1c0000gp/T/danger-results.json
```

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against cfa8fb80d2b65f4c4fa0b54d25352a3a0ff58f75
</p>

**Comment by SKY-DT-TEAMCITY** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Hello World</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against cfa8fb80d2b65f4c4fa0b54d25352a3a0ff58f75
</p>

**Comment by ashfurrow** [comment]
I'm locking this thread to prevent further comments from people testing Danger. Good luck to you all!

**Pull request merged**